src/components/Navigation/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as Navigation } from './Navigation';
```

Comment:
you only need to do this if there is a named export in the form of `export const Component`. Otherwise, `export Component from './'` is sufficient.